build.gradle

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+buildscript {
+  repositories {
+    jcenter()
+    maven { url "http://spinnaker.bintray.com/gradle" }
+    maven { url "https://plugins.gradle.org/m2/" }
+  }
+  dependencies {
+    classpath 'com.netflix.spinnaker.gradle:spinnaker-gradle-project:3.16.0'
+  }
+}
+
+description = "Redis embedded server for Java integration testing"
+group = "com.netflix.spinnaker.embedded-redis"
+apply plugin: 'spinnaker.project'
+apply plugin: 'groovy'
+
+//groovy only for tests
+tasks.compileGroovy.enabled = false
+
+spinnaker {
+  dependenciesVersion = "0.109.2"
+}
+
+test {
+  testLogging {
+    exceptionFormat = 'full'
+  }
+  minHeapSize = "512m"
+  maxHeapSize = "512m"
+}
+
+dependencies {
+  spinnaker.group('test')
```

Comment:
I suspect the `group('test')` and `testCompile groovy` aren't necessary here?

Reply:
I need some of those test dependencies (junit) - should I just bring in that one instead of all of the group 'test'?

Reply:
I'd probably just add junit, our test group brings in spock and friends which we don't need (for now anyway)

Reply:
Ok fixing!